src/test/java/se/sundsvall/emailsender/service/MicrosoftGraphMailSenderTests.java

```diff
@@ -88,7 +93,13 @@ void sendEmail() {
 		verify(mockUserItemRequestBuilder).sendMail();
 		verify(mockSendMailRequestBuilder).post(mockSendMailPostRequestBody);
 
-		verifyNoMoreInteractions(microsoftGraphMailSenderSpy, mockMessage, mockSendMailPostRequestBody, mockGraphServiceClient, mockUsersRequestBuilder, mockUserItemRequestBuilder, mockSendMailRequestBuilder);
+		verifyNoMoreInteractions(microsoftGraphMailSenderSpy);
+		verifyNoMoreInteractions(mockMessage);
```

Comment:
any particular reason to not lump them together?

Reply:
Damnit! I had them grouped, but split them up to be able to determine which one that made an extra, unexpected invocation. Will regroup them again